README.md

```diff
@@ -2,66 +2,226 @@
+
+If you specified a folder in the `files` list, we take all the content of the specified folder to put in the `targetFolder`.
+
+* `exclude`: String[], optional.
+
+This attribute is a list of RegExp to exclude some files from moving into the `targetFolder`.
+
+**Examples of metadata**:
+
+* For `githubator-plugin`:
+
 ```json
 {
-    "key": "terraform_webapp",
-    "name": "Web application",
-    "type": "project",
-    "description": "Schema of web application.",
-    "url": null,
-    "models": ["plugin1/model1", "plugin1/model2", "plugin2/model3"],
-    "files": ["plugin1/model1/main.js", "plugin1/model2/main.js", "plugin2/model3/app.js"]
+  "type": "model",
+  "plugin": "githubator-plugin",
+  "name": "Default JS front-end workflow",
+  "description": "Default JS front-end workflow",
+  "tags": ["CI", "GithubAction", "JS", "front-end"],
+  "icons": "/templates/DefaultWorkflowJS/icon.svg",
+  "schemas": ["/templates/DefaultWorkflowJS/schemas.svg"],
+  "files": ["/templates/DefaultWorkflowJS/CI-JS.yml"],
+  "targetFolder": ".github/workflows",
+  "exclude": []
 }
 ```
 
-- `key` attribute is the folder name of your template under the `templates` folder.
-- `name` attribute is the display name used in leto-modelizer.
-- `plugin` attribute is the plugin name used by the template. Only for `model` and `component` templates.
-- `type` attribute is the template type, either `model`, `component` or `project`.
-- `description` attribute is the description used in leto-modelizer to explain the usage of this template.
-- `url` attribute is an url attached to a documentation of this template used in leto-modelizer.
-- `models` attribute is the list of models used in the project. Same as `files`, without the file name. Only for `project` template.
-- `files` attribute is the list of files used in the template.
+Would move `/templates/DefaultWorkflowJS/CI-JS.yml` into `.github/workflows` in user folder.
 
-2. Create the folder of your template with the chosen key you provided in the [index.json](index.json)
-3. Create all the files (you) mentioned in this folder with the code of your template
-4. (Optional) Provide an icon for your template (only `icon.svg` will be used by leto-modelizer)
-5. (Optional) Provide a schema for your template (only `schema.svg` will be used by leto-modelizer)
+* For `terrator-plugin`:
+
+```json
+{
+  "type": "model",
+  "plugin": "githubator-plugin",
+  "name": "Default JS front-end workflow",
+  "description": "Default JS front-end workflow",
+  "tags": ["CI", "GithubAction", "JS", "front-end"],
+  "icons": "/templates/TerraformInfra/icon.svg",
+  "schemas": ["/templates/TerraformInfra/schema.svg"],
+  "files": ["/templates/TerraformInfra"],
+  "targetFolder": "terraform/infra",
+  "exclude": ["^*\\.svg$"]
+}
+```
+
+`/templates/TerraformInfra` contains: `icon.svg`, `schema.svg` and `main.tf`.
+
+Would move `/templates/TerraformInfra/main.tf` into `terraform/infra` in user folder.
```

Comment:
In this example, the file /templates/TerraformInfra/main.tf will be moved into `terraform/infra` in user folder. So you would have `terraform/infra/main.tf` etc

Reply:
done